session/avoid_session_start.rst

```diff
@@ -29,7 +29,7 @@ access the flash messages:
 
 .. code-block:: html+twig
 
-    {% if app.request.hasPreviousSession %}
+    {% if app.request.method == 'POST' %}
```

Comment:
I do not understand this change. I mean the flash entries will probably be created when processing a POST request, but the reason we register them is that we redirect at the end and we need to display them in response for the subsequent event which will be a GET request then. So with this change the flash messages are likely to never be shown.

Reply:
See https://github.com/symfony/demo/pull/859 for more details. People seem convinced that this is the right fix.

Reply:
Let me crosspost my comment there too then. :)